### PR TITLE
Add lottie

### PR DESCRIPTION
**Pull request by YKo20010**
Added lottie and placeholder animations to onboarding.

### CODE REVIEW

File: app/src/main/java/com/cornellappdev/android/eatery/model/enums/OnboardingPageType.java [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+            case MENUS:
+                return R.raw.lottie;
+            case COLLEGETOWN:
+                return  R.raw.lottie2;
+            case TRANSACTIONS:
+                return  R.raw.lottie3;
+            case LOGIN:
+                return  R.raw.lottie;
+            default:
+                return  R.raw.lottie;
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/model/enums/OnboardingPageType.java]: please rename these to something more descriptive. also couldn't you do something like 
```
case MENUS, LOGIN:
    return R.raw.lottie;
.....
```

File: app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingActivity.java [hunk 2]
```diff
@@ -25,6 +25,20 @@ protected void onCreate(Bundle savedInstanceState) {
         mViewPager = findViewById(R.id.onboarding_viewpager);
         mViewPager.setAdapter(new OnboardingPagerAdapter(getSupportFragmentManager()));
         mViewPager.setCurrentItem(0);
+        mViewPager.setOnPageChangeListener(new ViewPager.OnPageChangeListener() {
+            @Override
+            public void onPageScrolled(int position, float positionOffset, int positionOffsetPixels) {
+                // Must be overridden.
```
Comment (ningning621) [inline app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingActivity.java]: is this a `TODO` or an intentional empty method?

File: app/src/main/java/com/cornellappdev/android/eatery/model/enums/OnboardingPageType.java (hunk 1 above)
Comment (YKo20010) [reply app/src/main/java/com/cornellappdev/android/eatery/model/enums/OnboardingPageType.java]: image

File: app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingActivity.java (hunk 2 above)
Comment (YKo20010) [reply app/src/main/java/com/cornellappdev/android/eatery/onboarding/OnboardingActivity.java]: It's intentionally empty bc we don't have to do anything with it, but it must be overridden to compile.

**Pull request merged**